### DISCUSSION TITLE
Allow commenting on non-pull-request modified code

### DISCUSSION THREAD

**Discussion by jweg29** [community/community#8982] [Pull Requests]
During a pull request it would be great to be able to comment on lines that were not directly changed. For instance, if a line should be modified/removed or even to point out a side-effect of a change in the pull request. Other tools like Phabricator have supported this for years and it would be a great addition to Github. Thanks!

**Comment by xmo-odoo** [comment]
FWIW #9099 seems like a duplicate and apparently lucked out on visibility, so this should probably be closed and redirected over there or something.
